core/src/test/scala/integration/kafka/api/ConsumerTest.scala

```diff
@@ -237,7 +237,8 @@ class ConsumerTest extends IntegrationTestHarness with Logging {
 
   def testUnsubscribeTopic() {
     val callback = new TestConsumerReassignmentCallback()
-    this.consumerConfig.setProperty(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "100"); // timeout quickly to avoid slow test
+    this.consumerConfig.setProperty(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "100");
+    // timeout quickly to avoid slow test
```

Comment:
If we really want the comment to be on a new line, I think it should be before setting the property.